scss/partials/_utility.scss

```diff
@@ -127,21 +127,19 @@
   &__four {
     &--homepage-blog {
       @extend .grid-col-content__four;
-      padding-top: 5.0rem;
+      padding-top: 5rem;
 
       @include breakpoint(desktop) {
         padding-top: 0;
       }
     }
   }
 
-
-
   @include breakpoint(desktop) {
     &__two {
       @include grid();
       align-items: center;
-      column-gap: 16.0rem !important;
+      column-gap: 16rem !important;
```

Comment:
Do we know why we ended up needing an important here? Is there something upstream we could tweak to remove this need?

Reply:
...hmm it was there when I found it!? I summon @Wmiranda1 for a proper answer

Reply:
Sorry about that totally my fault and was catching up to speed on BEM. It was added because the grid mixin on line 140 has a value declared for column-gap. The simple fix is to remove the mixin and just make it `display: grid`

Reply:
@Wmiranda1 it's all good. Nothing major. I'll merge this branch and leave any change on this to you.